### PR TITLE
Update tag link

### PR DESCRIPTION
**Pull request by fmigot**
Fix proposal for this [issue #82 ](https://github.com/juliuscc/semantic-release-slack-bot/issues/82).

### CODE REVIEW

**Comment by juliuscc** [comment]
Thanks for the PR @fmigot!

**Review by juliuscc — APPROVED** [review]

File: lib/success.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ module.exports = async (pluginConfig, context) => {
                   type: 'mrkdwn',
                   text: `:package: *<${repo.URL}|${repo.path}>:*   <${
                     repo.URL
-                  }/releases/tag/${gitTag}|${gitTag}>`
+                  }/releases/${gitTag}|${gitTag}>`
```
Comment (juliuscc) [inline lib/success.js:109]: Does this work with GitHub and Bitbucket?

**Review by juliuscc — CHANGES_REQUESTED** [review]

File: lib/success.js (hunk 1 above)
Comment (fmigot) [reply lib/success.js]: Yes you're totally right! I've just made an update to manage **Gitlab** and **Github** tag url in your default template.
Nothing changes for Github projects, the tag url is still: `${repo.URL}/releases/tag/${gitTag}` and now for Gitlab projects, the url is: `${repo.URL}/-/releases/${gitTag}` ;)

I don't know what's the url for Bitbucket projects but if needed feel free to add a new condition later.

**Review by juliuscc — APPROVED** [review]
Great that you have added tests as well! I think this is ready to merge 👍

**Pull request merged**